lib/msal-common/src/error/AuthError.ts

```diff
@@ -7,11 +7,9 @@ import { Constants } from "../utils/Constants.js";
 import * as AuthErrorCodes from "./AuthErrorCodes.js";
 export { AuthErrorCodes };
 
-export const AuthErrorMessages = {
-    [AuthErrorCodes.unexpectedError]: "Unexpected error in authentication.",
-    [AuthErrorCodes.postRequestFailed]:
-        "Post request failed from the network, could be a 4xx/5xx or a network unavailability. Please check the exact error code for details.",
-};
+export function getDefaultErrorMessage(code: string): string {
+    return `See https://aka.ms/msaljs/common/errors#${code} for details`;
```

Comment:
This is fine for now but I think eventually it would be good to just have a single source of truth for all Errors

Reply:
Merged these two docs into one and placed under root /docs folder.